bodhi/tests/client/test___init__.py

```diff
@@ -1618,6 +1618,41 @@ def test_edit_with_errors(self, send_request):
         self.assertEqual(result.exit_code, 1)
         self.assertEqual(result.output, ("ERROR: an error was encountered... :(\n"))
 
+    @mock.patch('bodhi.client.bindings.BodhiClient.csrf',
+                mock.MagicMock(return_value='a_csrf_token'))
+    @mock.patch('bodhi.client.bindings.BodhiClient.send_request',
+                return_value=client_test_data.EXAMPLE_RELEASE_MUNCH, autospec=True)
+    def test_edit_critpath_pkgs(self, send_request):
+        """
+        Assert correct behavior while editing critpath_pkgs.
+        """
+        runner = testing.CliRunner()
+
+        result = runner.invoke(
+            client.edit_release,
+            ['--name', 'F27', '--critpath-pkgs', 'python,perl,php', '--url',
+             'http://localhost:6543', '--username', 'bowlofeggs', '--password', 's3kr3t'])
```

Comment:
You shouldn't need the ```--url```, ```--username```, or ```--password``` flags here, and removing them makes the test simpler and more targeted.

Reply:
Wouldn't we need ```--username``` and ```--password``` for authentication?
Test goes into forever wait if I don't provide those. (waiting on authentication maybe)